build_library/prod_image_util.sh

```diff
@@ -87,7 +87,8 @@ EOF
   # Make the filesystem un-mountable as read-write and setup verity.
   if [[ ${disable_read_write} -eq ${FLAGS_TRUE} ]]; then
     "${BUILD_LIBRARY_DIR}/disk_util" --disk_layout="${disk_layout}" \
-      verity "${BUILD_DIR}/${image_name}"
+      verity "${BUILD_DIR}/${image_name}" | \
+      tee "${BUILD_DIR}/verity_info"
```

Comment:
How about add support to disk_util for parsing the output of veritysetup and add an option to specify where to write the hash to. Something like:

```sh
disk_util --disk_layout="${disk_layout}" verity \
  --root_hash="${BUILD_DIR}/${image_name%.bin}_verity.txt" \
  "${BUILD_DIR}/${image_name}"
```

would create a file named `coreos_production_image_verity.txt` containing the simple hash:

```
bd4cf58d66bb3ffde9bcb65bbd6b82303034b181106aa83f521864a0c3cddbad
```

If you put those changes in a PR that doesn't include the experimental grub bit we can merge it any time. The grub changes will need to wait until the hash is in the kernel.